test/unit/operations/find.test.ts

```diff
@@ -40,7 +40,11 @@ describe('FindOperation', function () {
   describe('#execute', function () {
     context('command construction', () => {
       const namespace = ns('db.collection');
-      const server = new Server(new Topology([], {} as any), new ServerDescription(''), {} as any);
+      const server = new Server(
+        new Topology([], {} as any),
+        new ServerDescription('a:1'),
```

Comment:
that was kind of my thought, if empty string results in a malformed address, it would be better to guard against it with an actual error, otherwise it's not actually clear that we need to pass in something more legitimate looking and why it might be necessary, so it could trip up future developers

Reply:
Yea, I agree, added an if stmt, I blocked both nullish and empty string inputs, any remaining validation is up to HostAddress